### DISCUSSION TITLE
Lazer doesn't run in fullscreen exclusive

### DISCUSSION THREAD

**Discussion by smoogipoo** [ppy/osu#30523] [Q&A]
Originally posted by @ciinTri

---

### Type

Game behaviour

### Bug description

I'm having issue with lazer not running in Fullscreen Exclusive. 

Tried osu! stable and Fullscreen Exclusive seemed to work, the volume panel doesn't appear when adjusting it when the game is open. On lazer, the opposite happens with volume panel appearing leading to a significantly higher Input Latency compared to stable.
![volume panel appearing](https://github.com/user-attachments/assets/696a286d-9907-42fa-b21d-7e7dfd2b9257)

It also shows "Exclusive: unknown" when running Latency Certifier in lazer.
![exclusive unknown](https://github.com/user-attachments/assets/5816c5c8-fd62-4b58-920b-848e07fae201)

List of things I've tried
 - Disabling Fullscreen Optimizations in Compatibility
 - Direct3D11, OpenGL, Vulkan (Experimental), Direct3D11 (Experimental) Renderers
 - Alt + Enter to switch between windowed, borderless, fullscreen
 - Setting GameDVR_FSEBehavior, GameDVR_FSEBehaviorMode, and GameDVR_DSEBehavior to a value of 2 in Registry Editor
 - Setting the Flags value of the game in Computer\HKEY_CURRENT_USER\System\GameConfigStore\Children to 211 from 11 in Registry Editor
 - Deleting the Game's key in Computer\HKEY_CURRENT_USER\System\GameConfigStore\Children
 - Disable/Enable Game Mode
 - Disable/Enable Hardware Accelerated GPU Scheduling (HAGS)

### Version

Lazer 2024.1009.1

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17640078/compressed-logs.zip)

**Comment by smoogipoo** [comment]
Originally posted by @peppy

---

Fullscreen &quot;exclusive&quot; is no longer a thing with direct3d 11. It&#39;s not required to achieve lower latency.

**Comment by Noxturn3** [comment]
I am having the exact same issue listed here and as far as I am aware FSE is still suported by dx11; it is dx12 that drops support for it in favor of FSO. I personally would like to see if this can be resolved as FSO makes any game feel very weird for me.
